.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -76,6 +78,9 @@ parameters:
 - name: ENV_NAME
   description: ClowdEnvironment name (ephemeral, stage, prod)
   required: true
+- name: PROMETHEUS_PUSHGATEWAY_URL
```

Comment:
The parameter key for the provided value is `PROMETHEUS_PUSHGATEWAY` so the same key needs to be here, and then line 65 would need to refer to that key name as well.

Reply:
changed.